client/app/scripts/superdesk-search/views/item-sortbar.html

```diff
@@ -9,7 +9,7 @@
 	        </li>
 	    </ul>
 	</div>
-	<button ng-click="toggleDir()" class="pull-left direction" title="{{active.dir}}">
+	<button ng-click="toggleDir()" class="pull-left direction" tooltip="{{ :: active.dir}}" tooltip-placement="bottom">
```

Comment:
Just to be sure - the value of the `active.dir` expression does not change in the lifetime of the view?